### PR TITLE
Deploy WNP messages for 82

### PR DESCRIPTION
**Pull request by piatra**

### CODE REVIEW

**Comment by piatra** [comment]
I've excluded the badge for `fr` locales because the media keys message does not show up for `fr`. The remaining message (DOH) is only available to `US` geo so we would risk adding a badge where actually nothing new is showing.

File: whats-new-panel.yaml [hunk 1]
```diff
@@ -174,7 +96,42 @@
     || (messageImpressions['WHATS_NEW_BADGE_82']|length >= 1 && currentDate|date -
     messageImpressions['WHATS_NEW_BADGE_82'][0] <= 345600000)) &&
     "browser.messaging-system.whatsNewPanel.enabled"|preferenceValue &&
-    region == "US"
+    localeLanguageCode != 'fr'  # Because WNP_MEDIA message is not yet translated
   template: toolbar_badge
   trigger:
     id: toolbarBadgeUpdate
+- id: WHATS_NEW_DNS_OVER_HTTPS
```
Comment (ncloudioj) [inline whats-new-panel.yaml:103]: Normally, we'd end the id with a browser version, shall we stick to that convention?

File: whats-new-panel.yaml [hunk 2]
```diff
@@ -174,7 +96,42 @@
...
+  template: whatsnew_panel_message
+  order: 2
+  content:
+    bucket_id: WHATS_NEW_DNS_OVER_HTTPS
```
Comment (ncloudioj) [inline whats-new-panel.yaml:107]: Ditto.

**Review by ncloudioj — APPROVED** [review]
r+ with comments.

Thanks!

**Pull request merged**